app/views/layouts/base.html.erb

```diff
@@ -18,6 +18,7 @@
     <%= favicon_link_tag asset_path('images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
 
     <%= stylesheet_link_tag "application" %>
+    <link rel="stylesheet" href="/location-autocomplete.min.css" />
```

Comment:
Could this be included via a Sass `@import` statement in our main `application.scss`?

That way we don't have to copy it to the public folder and keep it in version control.

There are some caching benefits to having it it separate however. If we go this route, I'd prefer to have it as a static / unminified CSS file in our `app/assets/stylesheets` folder, and `stylesheet_link_tag` to it as usual so that `propshaft` picks it up and fingerprints it.

Combining both thoughts above could mean an `accessible-autocomplete.scss` in our stylesheets folder, that is just a single line import of the CSS from the node bundle, but fingerprinted through `propshaft` and I think is "best of both worlds" and truly a premature optimisation for the ages. :rocket: